index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body onload="featuredClubs()">
+  <div class="mobile-menu">
+    <div class="mobile-menu-background">
+      <img src="images/Iconcross-btn.png" alt="Closing button mobile" class="cross-button pointer">
+    </div>
+    <ul class="mobile-menu-list">
+      <li class="mobile-menu-listItems"><a href="index.html">Home</a></li>
+      <li class="mobile-menu-listItems"><a href="about.html">About</a></li>
+      <li class="mobile-menu-listItems"><a href="#main-program">Program</a></li>
+      <li class="mobile-menu-listItems"><a href="#clubs">Clubs</a></li>
+      <li class="mobile-menu-listItems"><a href="#partners">Partners</a></li>
+      <li class="mobile-menu-listItems"><a href="#contact">Contact</a></li>
+    </ul>
+  </div>
+
+  <header>
+    <div class="header1 flex">
+      <ul class="header1-list flex pointer">
+        <li class="header1-partners-itemss"><i class="fa fa-facebook-f"></i></li>
+        <li class="header1-partners-itemss"><i class="fa fa-twitter"></i></li>
+        <li class="header1-partners-itemss">English</li>
+        <li class="header1-partners-itemss">My Page</li>
+        <li class="header1-partners-itemss">Logout</li>
+      </ul>
+    </div>
+    <div class="header2 flex">
+      <div class="header2-inner flex">
+        <div class="logo">
+          <a href="index.html"><img src="images/small_logo.png" alt="Program Logo"></a>
+        </div>
+        <nav class="nav-bar flex pointer">
+          <a href="about.html">
+            <li class="hover-effect">About</li>
+          </a>
+          <a href="#main-program">
+            <li class="hover-effect">Program</li>
+          </a>
+          <a href="#clubs">
+            <li class="hover-effect">Clubs</li>
+          </a>
+          <a href="#partners">
+            <li class="hover-effect">Partners</li>
+          </a>
+          <a href="#contact">
+            <li class="hover-effect">Contact</li>
+          </a>
+        </nav>
+        <nav class="union pointer">
+          <img src="images/Union.png" alt="Mobile Menu Option">
+        </nav>
+      </div>
+    </div>
+  </header>

```

Comment:
- Currently, I think the header of your mobile version currently contains the top dark bar and a bottom white bar that is only meant to be shown on your desktop screen. Apparently, from the Behance design, you only need just the **hamburger menu** on the **top right**.

Here is what I mean 👇🏽 
| Your mobile header  |Behance mobile header |
| :---:   | :---: | 
|![badmob](https://user-images.githubusercontent.com/99246359/203646395-524e560b-8f08-4e81-ac6d-8fc396147ed6.PNG)  |![goodmob](https://user-images.githubusercontent.com/99246359/203646486-1d00daf6-7132-4b2d-8211-4cae6881c7f2.PNG)  |

- Why don't we refactor your mobile header to look just like Behance? 🌸 
You can simply hide them on mobile and show them in your desktop version(using media query)